package.json

```diff
@@ -12,7 +12,8 @@
     "coverage": "node npm/coverage.js",
     "dev:dashboard": "nodemon ./dashboard/server/index.js",
     "dev:client": "next ./frontend",
-    "build": "node npm/build.js"
+    "build": "node npm/build.js",
+    "generate-docs": "jsdoc --configure jsdoc.conf.json -d ./out"
```

Comment:
We can maybe remove the `generate-docs` command from this file itself since there are no source files? Another solution could be to generate docs for each of the child subdirectories.

See `test:unit` for example.

Reply:
I am not getting this change will you please elaborate on it.